src/streaming/MediaPlayerEvents.js

```diff
@@ -105,6 +105,10 @@ class MediaPlayerEvents extends EventsBase {
          * @event MediaPlayerEvents#PERIOD_SWITCH_STARTED
          */
         this.PERIOD_SWITCH_STARTED = 'streamswitchstarted';
+
+        this.QUALITY_CHANGE_START = 'qualityChangeStart';
```

Comment:
Yes they are all public facing.  All events in MediaPlayerEvents are public.  Any event in CoreEvent or Protection marked with "public_" will be moved into this class for public access via MediaPlayerEvents,  and all events, core, public, protection etc... are aggregated into Events.js for internal consistent access .   Not sure I follow the comment that events in the client are lower case?